### PR TITLE
Fix several init bugs and improve the docs

### PR DESCRIPTION
**Pull request by adam-hart**
Here are some of the things that I found while re-following the on-boarding process this afternoon.  It should make life a little easier.

1. Adds better documentation on how to set up an administrator Firebase user.
2. Pushes the database rules before requiring a full Firebase deploy.  This is one of the issues Scott reported.
3. Adds some seed taxonomies to allow locations to actually be added.  They error out without any taxonomies present, and we have no UI for that yet.

@zendesk/volunteer

### CODE REVIEW

File: docs/MANAGE.md [hunk 1]
```diff
@@ -39,6 +39,13 @@ Services contain basic information such as a name, description, and category, as
 
 ### How to change your data
 
+#### Set up a Firebase user
+
+1. Navigate to the Authentication page of your [Firebase Console](https://firebase.google.com/console).
+2. Make sure you have Email/Password authentication enabled.  It should prompt you to enable it if you hover over *ADD USER*, otherwise you can find the setting under the *SIGN-IN METHOD* tab near the top of the page.
```
Comment (craig-day) [inline docs/MANAGE.md]: you can make these all `1.` and they will get evaluated when rendered, makes changing instructions nicer.

File: docs/SETUP.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Link-SF is designed to be free to setup and free to run. Since hosting files is
 
 * [Firebase](https://firebase.google.com/console):  Firebase is the only required account, and it's completely free.  It will provide everything we need to get up and running, including database storage, authentication, and hosting.
 
-  **Note**  Every administrator on your account will need to be created under the Authentication panel in your new Firebase Console.  You won't be able to adjust your production data from the app without it!
+  **Note**  Every administrator on your account will need to be created under the Authentication panel in your new Firebase Console.  You won't be able to adjust your production data from the app without it!  For information on how to set up users, see the [Management documentation](https://github.com/zendesk/linksf/blob/master/docs/MANAGE.md) under *Set up a Firebase user*.
```
Comment (craig-day) [inline docs/SETUP.md]: can link directly to the section with 

```
linksf/blob/master/docs/MANAGE.md#set-up-a-firebase-user
```

notice its also a relative link so you don't need the full https foobar

File: docs/SETUP.md [hunk 3]
```diff
@@ -64,7 +56,7 @@ In the project root, you will find three configuration files:
 
 ### Install tools
 
-If you are working from a Unix system with Bash, you can use the pre-written boostrap script, which will complete all remaining steps in this doc.
+If you are working from a Unix system with Bash, you can use the pre-written boostrap script.  If you do, you can skip to [Database Population](#database-population).
```
Comment (craig-day) [inline docs/SETUP.md]: wording seems a little confusing. Maybe something like 

> bootstrap script and then proceed with the database population step.

**Comment by adam-hart** [comment]
Updated @craig-day

**Pull request merged**